docs/advanced/releases.md

```diff
@@ -50,31 +50,39 @@ Release Candidates are the same as beta versions. They should be ready to go to
 
 If this is a **Release Candidate version** release, the steps to follow are:
 
-1. Go to the release branch `git checkout release/x.y.z`.
+1. Go to the release's branch `git checkout release/x.y-stable`.
 1. Update `.decidim-version` to the new version `x.y.z.rc1`
 1. Run `bin/rake update_versions`, this will update all references to the new version.
 1. Run `bin/rake bundle`, this will update all the `Gemfile.lock` files
 1. Run `bin/rake webpack`, this will update the JavaScript bundle.
-1. Commit all the changes: `git add . && git commit -m "Bump to rcXX version" && git push origin release/x.y.z`.
-1. Follow the link resulting from the previous command to create the PR for the new version. The base branch of the PR must be `master`.
+1. Commit all the changes: `git add . && git commit -m "Bump to rcXX version" && git push origin release/x.y-stable`.
+1. Tag the release candidate with `git tag vX.Y.Z && git push origin vX.Y.Z`.
 1. Usually, at this point, the release branch is deployed to meta-decidim during, at least, one week to validate the stability of the version.
 
 ### During the validation period
 
 1. During the validation period bugfixes will go directly to the current `release/x.y.z` branch and ported to `develop`.
 1. During the validation period, translations to the officially supported languages must be added to Crowdin and when done, merged into `release/x.y.z`.
 
-### Major/Minor versions
+## Major/Minor versions
 
 Release Candidates will be tested in a production server (usually meta-decidim) during some period of time (a week at least), when they are considered ready, it is time for them to be merged into `master`:
 
-1. Go to the release branch `git checkout release/x.y.z`.
+1. Go to the release branch `git checkout release/x.y-stable`.
 1. Update `.decidim-version` by removing the `.rcN` suffix, leaving a clean version number like `x.y.z`
 1. Run `bin/rake update_versions`, this will update all references to the new version.
 1. Run `bin/rake bundle`, this will update all the `Gemfile.lock` files
 1. Run `bin/rake webpack`, this will update the JavaScript bundle.
-1. Commit all the changes: `git add . && git commit -m "Bump to v0.XX.0 final version" && git push origin release/x.y.z`.
-1. Follow the link resulting from the previous command to create the PR for the new version in case it has not been created already, but still don't merge it. The base for the PR should be `master`.
+1. Commit all the changes: `git add . && git commit -m "Bump to v0.XX.0 final version" && git push origin release/x.y-stable`.
+1. Create the PR for the new version.
+    1. `git checkout master && git checkout -b release/x.y.z`
+    1. `git merge release/x.y-stable`
+    1. `git checkout --theirs *`
+    1. `git checkout --theirs .github/* \.*`
+    1. Review changes in `CHANGELOG.md` and manually update it if required.
```

Comment:
```suggestion
    1. Review changes in `CHANGELOG.md` and manually update and create a "changelog" commit it if required.
```
If we modify the changelog and commit here, should this be commited to `release/x-y-stable` too?

Reply:
@microstudi the suggested line reads weird, there's an extra "it" around!

Reply:
uups, thanks!

Reply:
@microstudi Changelog in release/x-y-stable and master should be exactly the same at this point but automatic merges use to leave some inconsistencies so the changelog in master must be checked, the one in release/x-y-stable should not be modified.